Makefile

```diff
@@ -1,7 +1,55 @@
-run-tests:
```

Comment:
Here is one issue: Since MODULE is not set in the root make file, I cannot do `make run-tests` in the root. This is a requirement to execute all tests from the main OpenSlides repo (each service must have a run-test target in the root make file).

Also I cannot do other targets as well. I would expect, that if there is a makefile (and more over the top level one), all targets should work. I would propose to move this makefile to shared, let writer (and later the reader) use the shared Makefile. The shared makefile has the MODULE set to shared per default, which is overwritten by the writer and reader.

The root makfile just executes all tests in the subdirectories.

Reply:
Ah yes, I noticed that as well but put it on the backlog to fix it later. The approach to move the makefile code to shared folder doesn't work very well because the handling of all targets can't be as generic as now since the relative paths to reader/writer and shared differ then.
I adjusted the main makefile now and just defined different targets depending on whether `MODULE` is defined or not, this should suite your requirements.